### ISSUE TITLE
Provide SHELL Keyword from Dockerfile

### ISSUE THREAD

**Issue by SebBaeck** [marcuslonnberg/sbt-docker#84]
It should be possible to change the executing shell, like the SHELL command from DOCKERFILE: https://docs.docker.com/engine/reference/builder/#shell

**Comment by marcuslonnberg** [comment]
Do you mean in some other way than what is supported now?
https://github.com/marcuslonnberg/sbt-docker/blob/master/src/main/scala/sbtdocker/DockerfileCommands.scala#L243

**Comment by SebBaeck** [comment]
Yeah. If I'm right, https://github.com/marcuslonnberg/sbt-docker/blob/master/src/main/scala/sbtdocker/DockerfileCommands.scala#L243 is also executed in /bin/sh, see: https://github.com/marcuslonnberg/sbt-docker/blob/8a6afaceb362542b7d64538d48699ff3c361591f/src/main/scala/sbtdocker/Instructions.scala#L133

The SHELL command from Dockerfile: https://docs.docker.com/engine/reference/builder/#shell could override the default shell, used by shell form commands, like RUN.

An example command from my Dockerfile:
SHELL [ "/bin/bash", "-c" ]

I would like to create a pull request for the required changes to the files: "DockerfileCommands.scala" and "Instructions.scala" but actually I can't do that.
